biomeidentifier.cpp

```diff
@@ -113,7 +114,7 @@ QColor BiomeInfo::mixColor( QColor colorizer, QColor blockcolor )
   }
 //float lig = std::clamp( ligC + ligB - 0.5f, 0.0f, 1.0f );  // more brightness contrast
   float lig = (ligC + ligB ) / 2.0f;
-  return QColor::QColor().fromHslF( hue, sat, lig );
+  return QColor().fromHslF( hue, sat, lig );
```

Comment:
The `fromHslf` function is static, so why not use the simple `QColor::fromHslf(...)`?